sdk/program/src/program.rs

```diff
@@ -64,3 +64,42 @@ extern "C" {
         signers_seeds_len: u64,
     ) -> u64;
 }
+
+#[allow(unused)]
+const MAX_RETURN_DATA: usize = 1024;
```

Comment:
Remove the attribute and make this `pub`?

Reply:
Also add doc comment

Reply:
Fixed.